lib/client.js

```diff
@@ -216,6 +224,34 @@ class ServiceClient {
             timeout: DEFAULT_REQUEST_TIMEOUT
         }, this.options.defaultRequestOptions);
 
+        this.options.retryOptions = Object.assign({
+            retries: 0,
+            factor: 2,
+            minTimeout: 200,
+            maxTimeout: 400,
+            randomize: true,
+            // By default all errors are indicated as being transient
+            // users should define their own check in order to handle errors as they require
+            // it takes the error as an argument
+            transientErrorCheck(err) {
+                // by default don't retry if the circuit breaker is open
+                if (err.type === ServiceClient.CIRCUIT_OPEN) {
```

Comment:
I thought this would be a good idea to have by default, but again if you just want it to return true by default I can remove this

Reply:
How about renaming this as `shouldRetry`? Open circuit breaker is also literally transient.

Reply:
In relation to this naming, it would conflict with here: https://github.com/zalando-incubator/perron/pull/33/files#diff-50cfa59973c04321b5da0c6da0fdf4feR314

What would your naming preference be for the already defined `shouldRetry`

Reply:
The existing `shouldRetry` is just an internal variable, which matters less than public API. Now I think that the existing one is a result of `retry` module's decision. So how about `retrying`, `retryingNext` or `willRetry`?

Reply:
Good point, will update